lib/blockTemplate.js

```diff
@@ -61,6 +67,12 @@ var BlockTemplate = module.exports = function BlockTemplate(
     masternodeReward = rpcData.payee_amount;
     masternodePayee = rpcData.payee;
     masternodePayment = rpcData.masternode_payments;
+    zelnodeBasicAddress = rpcData.basic_zelnode_address;
```

Comment:
Could you wrap these setters in an `if (coin.payZelnodeRewards === true)` check? This would allow pool owners to update early and then enable zelnode payments at the time of the spork.

Reply:
implemented with latest commit